packages/ifixit-api-client/index.tsx

```diff
@@ -60,13 +60,21 @@ export class IFixitAPIClient {
    }
 
    async fetch(endpoint: string, statName: string, init?: RequestInit) {
+      const pskToken = process.env.DEV_API_AUTH_TOKEN;
+      const authHeader: { Authorization?: string } = pskToken
```

Comment:
In this case, isn't this type implicit in the expression?

Reply:
Here's the TypeScript error if that type's not included:
```
node_modules/@ifixit/analytics/node_modules/@ifixit/cart-sdk/node_modules/@ifixit/ifixit-api-client/index.tsx:72:16 - error TS2322: Type '{ Authorization: string; } | { length: number; toString(): string; toLocaleString(): string; pop(): [string, string] | undefined; push(...items: [string, string][]): number; concat(...items: ConcatArray<...>[]): [...][]; concat(...items: ([...] | ConcatArray<...>)[]): [...][]; ... 29 more ...; Authorization: string;...' is not assignable to type 'HeadersInit | undefined'.
  Type '{ Authorization?: undefined; }' is not assignable to type 'HeadersInit | undefined'.
    Type '{ Authorization?: undefined; }' is not assignable to type 'undefined'.

72                headers: {
                  ~~~~~~~
```

I'm guessing the problem is that the empty object doesn't get inferred to be `{Authorization?: string}` with `Authorization` unset. Instead, it gets picked up as `{}`, and then we access the unknown property `Authorization` on it, getting `undefined`. That's my best guess; it seems like something better inference could eliminate, but perhaps TypeScript's not able to handle that yet.